tests/benchmarks/test_h2o.py

```diff
@@ -1,36 +1,46 @@
 """
 h2o-ai benchmark groupby part running on coiled.
 """
+import os
 
-import dask
 import dask.dataframe as dd
 import pandas as pd
 import pytest
-from packaging.version import Version
 
-from ...utils_test import run_up_to_nthreads
+from ..utils_test import run_up_to_nthreads
+
+DATASETS = {
+    "0.5 GB (csv)": "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_single.csv",
+    "5 GB (csv)": "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_single.csv",
+    "50 GB (csv)": "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_single.csv",
+    "0.5 GB (parquet)": "s3://coiled-datasets/h2o-benchmark/N_1e7_K_1e2_parquet/*.parquet",
+    "5 GB (parquet)": "s3://coiled-datasets/h2o-benchmark/N_1e8_K_1e2_parquet/*.parquet",
+    "50 GB (parquet)": "s3://coiled-datasets/h2o-benchmark/N_1e9_K_1e2_parquet/*.parquet",
+}
+
+enabled_datasets = os.getenv("H2O_DATASETS")
+if enabled_datasets is not None:
+    enabled_datasets = {k.strip() for k in enabled_datasets.split(",")}
+    if unknown_datasets := enabled_datasets - DATASETS.keys():
+        raise ValueError("Unknown H2O dataset(s): ", unknown_datasets)
+else:
+    enabled_datasets = {"0.5 GB (csv)", "0.5 GB (parquet)", "5 GB (parquet)"}
```

Comment:
This is what is used by the non-A/B tests.

@fjetter I think there is *some* value in leaving the 0.5 GB datasets enabled in the general tests. Namely, it shows potential regressions in CSV reading, and it helps appreciating the gap in read speed of parquet vs csv.
Happy to change this default if you have strong opinions though.

Reply:
I believe there are plenty of benchmarks out there that show the diff between parquet and CSV.
I'm OK keeping them around if they are not too noisy/flaky